### PR TITLE
added drums and added useRef hooks for synth and synthtype

### PR DESCRIPTION
**Pull request by darshkpatel**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -78,46 +82,59 @@ function App() {
...
+        'loop 2': 'https://tonejs.github.io/audio/drum-samples/breakbeat.mp3'
+      }
+      if (e.target.innerText === 'loop 1' || e.target.innerText === 'loop 2' || e.target.innerText === 'loop 3' || e.target.innerText === 'loop 4') {
+        console.log(loopLinks[e.target.innerText])
+        const player = new Tone.Player(loopLinks[e.target.innerText]).toDestination();
+        player.loop = true;
+        player.autostart = true;
+      }
+      else {
+        const drum = new Tone.Player(`https://oramics.github.io/sampled/DM/CR-78/samples/${e.target.innerText}.wav`).toDestination();
```
Comment (darshkpatel) [inline src/App.jsx:116]: Don't link to binary files from a user's GitHub repo, put them in the project repo along with other static files 
Vercel serves files under 5mb  @codevaam

**Review by darshkpatel — COMMENTED** [review]
@codevaam

File: src/App.jsx [hunk 2]
```diff
@@ -78,46 +82,59 @@ function App() {
...
-          console.log('Initial Trigger: ', e.target.innerText, ' Final Trigger: ', playNote);
-        } catch (err) {
-          console.log(err);
+    const playNote = (e) => {
+      // Grabs note name from 'data-note'
+      console.log(e.touches.length);
+      try {
+        // synth.triggerAttack(e.target.innerText, '16n');
+        console.log('ran');
+        navigator.vibrate(10000);
```
Comment (darshkpatel) [inline src/App.jsx:92]: remove these console logs too , might help with the vibration being delayed

File: src/App.jsx (hunk 1 above)
Comment (darshkpatel) [reply src/App.jsx:116]: PWA should run offline without external dependencies. Having files in the app itself will help.

Comment (codevaam) [reply src/App.jsx]: Made the changes, check once and merge

**Pull request merged**